src/components/MdTable/MdTable.vue

```diff
@@ -296,6 +312,10 @@
       // wait for `selectingMode` from `TableRow`
       await this.$nextTick()
       this.syncSelectedValue()
+      if (this.asyncUrl) {
+        const data = (await axios.get(this.asyncUrl, { params: this.asyncParams, headers: this.asyncHeaders })).data
```

Comment:
Should there be a loading UI while requesting?
How about a named slot with indeterminate loading bar as default?